pure.zsh

```diff
@@ -331,6 +331,16 @@ prompt_pure_async_git_arrows() {
 	command git rev-list --left-right --count HEAD...@'{u}'
 }
 
+prompt_pure_async_renice() {
+	setopt localoptions noshwordsplit
+	if command -v renice >/dev/null; then
+		command renice +15 -p $$;
+	fi
+	if command -v ionice >/dev/null; then
+		command ionice -c 3 -p $$;
```

Comment:
Let's remove the trailing semi-colons here (they serve no purpose).

Reply:
Done